### PR TITLE
Move dashboard services to axios.

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: src/server/api/router.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ export default function apiRouter() {
         terrainHandler({
             method: "GET",
             pathname: "/dashboard",
+            headers: {
+                "Content-Type": "application/json",
+            },
```
Comment (ash3rz) [inline src/server/api/router.js:35]: Is this actually needed for a GET request?

Comment (sri2k1us) [reply src/server/api/router.js:35]: @johnworth had it in the fetch-request. So I copied it here.

**Comment by sri2k1us** [comment]
> Would it be possible to rename serviceFacade/dashboardServiceFacade.js to just serviceFacade/dashboard?

Done!

**Comment by sri2k1us** [comment]
Thanks for the reviews.

File: src/server/api/router.js (hunk 1 above)
Comment (psarando) [reply src/server/api/router.js:35]: I think our Swaggerized terrain endpoints use this header to decide what format to use in the response. So this content-type tells terrain we want a JSON response, but requesting `application/x-yaml` content-type will return a YAML response.

**Pull request merged**